diff --git a/lib/dev-ip.js b/lib/dev-ip.js
index 7f5500f60fe5996241641da6178f78898c0f6c2b..8bb084cb511a712f54ba9b18bb6e4cb4418e41b3 100644
--- a/lib/dev-ip.js
+++ b/lib/dev-ip.js
@@ -37,8 +37,8 @@ module.exports = getIp;
 if (require.main === module) {
 
     var out = getIp();
-    if (!out.length) {
-        return console.log(messages.error);
-    }
-    console.log(getIp("cli"));
+    if (out.length)
+        console.log(getIp("cli"));
+    else
+        console.log(messages.error);
 }
\ No newline at end of file
